prow/pluginhelp/pluginhelp.go

```diff
@@ -22,35 +22,35 @@ package pluginhelp
 // This includes repo specific configuration for every repo that the plugin is enabled for.
 type PluginHelp struct {
 	// Description is a description of what the plugin does and what purpose it achieves.
-	Description string
+	Description string `json:"description,omitempty"`
```

Comment:
These annotations seem unnecessary, especially since you're using CamelCase in the javascript. Default serialization should work.

Reply:
Done.